tests/test_cora_transform.py

```diff
@@ -20,109 +20,109 @@ class FormatTests(unittest.TestCase):
     """
 
     def test_check_yesno(self):
-        self.assertTrue(CORATransformer.Format.yesno.value.match("yes"))
-        self.assertTrue(CORATransformer.Format.yesno.value.match("no"))
-        self.assertFalse(CORATransformer.Format.yesno.value.match("don't know"))
-        self.assertFalse(CORATransformer.Format.yesno.value.match("Yes"))
-        self.assertFalse(CORATransformer.Format.yesno.value.match("No"))
-        self.assertFalse(CORATransformer.Format.yesno.value.match("Don't know"))
+        self.assertTrue(CORATransformer._Format.yesno.value.match("yes"))
```

Comment:
I'm a bit confused for the need of using the underscore, does it make a difference?

Reply:
The only difference it makes is to tell the reader that the function should not normally be called outside of the class . I.e you should normally treat it as private